test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -127,6 +127,16 @@ public void SetManualSasProvisioning(string hubHostname, Option<string> parentHo
             this.SetAuth(keyName);
         }
 
+        public void SetImageGarbageCollection(int minutesUntilCleanup)
+        {
+            this.config[Service.Edged].Document.ReplaceOrAdd("image_garbage_collection.enabled", true);
+            this.config[Service.Edged].Document.ReplaceOrAdd("image_garbage_collection.cleanup_recurrence", "1d");
+            this.config[Service.Edged].Document.ReplaceOrAdd("image_garbage_collection.image_age_cleanup_threshold", "10s");
+            TimeSpan two_minutes = new TimeSpan(0, 0, minutesUntilCleanup, 0);
```

Comment:
This variable is called `two_minutes`, but the actual number of minutes it represents is variable (the caller happens to pass in the hard-coded value `2`). We should update the variable name, or inline the TimeSpan ctor on the next line.

Reply:
Fixed.